web/js/components/layer/product-picker/browse/recent-layers.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Button, Tooltip } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faQuestionCircle } from '@fortawesome/free-regular-svg-icons';
+
+import SearchLayerList from '../search/search-layers-list';
+import LayerMetadataDetail from '../search/layer-metadata-detail';
+import {
+  recentLayerInfo,
+} from '../../../../modules/product-picker/util';
+import {
+  clearRecentLayers as clearRecentLayersAction,
+} from '../../../../modules/product-picker/actions';
+
+function RecentLayersList(props) {
+  const {
+    clearRecentLayers,
+    selectedLayer,
+    smallView,
+    isMobile,
+    recentLayers,
+  } = props;
+
+  const [tooltipVisible, toggleTooltip] = useState(false);
+
+  return (
+    <>
+      {!isMobile && !!recentLayers.length && (
+      <div className="recent-layers-header">
+        <h2> Recently Used Layers </h2>
+        <Tooltip
+          className="facet-tooltip-content"
+          isOpen={tooltipVisible}
+          target="recent-layer-tooltip-target"
+          placement="bottom"
+          toggle={() => toggleTooltip(!tooltipVisible)}
+          delay={{ show: 0, hide: 300 }}
+        >
+          <p>{recentLayerInfo}</p>
+        </Tooltip>
+        <FontAwesomeIcon
```

Comment:
Having the question mark hover tooltip as a reusable util could be helpful. This icon could also benefit from an on-hover `cursor: help` if it works for all browsers.